modules/rbac-overview.adoc

```diff
@@ -144,6 +144,10 @@ local role bindings, users, groups and service accounts are illustrated below.
 
 image::rbac.png[{product-title} RBAC]
 
+[WARNING]
+====
+The permissions `get pods/exec`, `get pods/*`, and `get *` can grant execution permissions to users. Ensure to apply only the permissions with the least privilege. For more information, see link:https://access.redhat.com/solutions/6989997[RBAC rules allow execution privileges].
```

Comment:
Based on the definitions in the SSG, I think your use of [_permission_](https://redhat-documentation.github.io/supplementary-style-guide/#privilege) and [_privilege_](https://redhat-documentation.github.io/supplementary-style-guide/#privilege) is inconsistent. And based on the diagram and the YAML example in the KCS article, I think the appropriate term for `get` etc is _rules_, not permissions. Also, the Kubernetes docs include a good definition of the [rule of least privilege](https://kubernetes.io/docs/concepts/security/rbac-good-practices/#least-privilege). 

```suggestion
The `get pods/exec`, `get pods/*`, and `get *` rules grant execution privileges when they are applied to a role. Apply the principle of least privilege and assign only the minimal RBAC rights required for users and agents. For more information, see link:https://access.redhat.com/solutions/6989997[RBAC rules allow execution privileges].
```
I am not confident that the wording in this suggestion is correct. Given the security implications, and that these terms are often used interchangibly in casual conversation, I think this would benefit from running this by a SME. 

/remove-label merge-review-in-progress
/remove-label merge-review-needed

Reply:
Calling these permission or rules are both fine for me. But original "The permissions ... can grant ... permissions" is definitely not good sentence using two words of "permissions". Thus "rules" and the suggested wording can be better because immediately before the sentence is some picture referencing "Rule".

Reply:
Thank you very much @michaelryanpeter @xingxingxia for your inputs :)

I addressed the change.

Reply:
Putting the PR for the merge review. Thank you again!!